From 3dec9bb3b6001015635c4898ddc9b11d5da4682b Mon Sep 17 00:00:00 2001
From: David Holsgrove <david.holsgrove@petalogix.com>
Date: Tue, 18 Oct 2011 10:12:12 +1000
Subject: [PATCH 12/79] gcc/config/microblaze/microblaze.h: Drop lgloss from LIB_SPEC

Following merge of xilinx newlib code into newlib repo, the merged
library is now libxil.a, not libgloss.a

lgloss-linux will be added through elf-gloss-linux.specs during dejagnu
testing

Signed-off-by: David Holsgrove <david.holsgrove@petalogix.com>
---
 gcc/config/microblaze/microblaze.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/gcc/config/microblaze/microblaze.h b/gcc/config/microblaze/microblaze.h
index 55df0b4..8b796b8 100644
--- a/gcc/config/microblaze/microblaze.h
+++ b/gcc/config/microblaze/microblaze.h
@@ -916,8 +916,8 @@ do {									 \
 #undef LIB_SPEC
 #define LIB_SPEC \
 "%{!nostdlib: \
-%{pg:-start-group -lxilprofile -lgloss -lxil -lc -lm -end-group } \
-%{!pg:-start-group -lgloss -lxil -lc -lm -end-group }} "
+%{pg:-start-group -lxilprofile -lxil -lc -lm -end-group } \
+%{!pg:-start-group -lxil -lc -lm -end-group }} "
 
 #undef  ENDFILE_SPEC
 #define ENDFILE_SPEC "crtend.o%s crtn.o%s"
-- 
1.7.1

